curiouslearning/FeedTheMonsterJS
# Fix Cr 65 Oromo language build

# Changes
- oromo language folder

# How to test
- Check the oromo language from dropdown

Ref: [CR-65](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-65)


[CR-65]: https://curiouslearning.atlassian.net/browse/CR-65?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
